### PR TITLE
add UserSettings to UserPref

### LINKED ISSUE #47: Change UserPrefs class in Model Component

### PR DESCRIPTION
**Pull request by Cronyxx**
Closes #47

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/UserSettings.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package seedu.address.commons.core;
```
Comment (shiyao821) [inline src/main/java/seedu/address/commons/core/UserSettings.java:1]: Nice, simple design!

File: src/main/java/seedu/address/commons/core/UserSettings.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+/**
+ * A Serializable class that contains the User settings.
+ * Guarantees: immutable.
+ */
+public class UserSettings implements Serializable {
+
+    private static final int DEFAULT_LOAN_PERIOD = 14;
+    private static final int DEFAULT_RENEW_PERIOD = 14;
+    private static final double DEFAULT_FINE_PERCENTAGE_INCREMENT = 10;
```
Comment (hoholyin) [inline src/main/java/seedu/address/commons/core/UserSettings.java:14]: Good use of constants here!

**Review by hoholyin — COMMENTED** [review]
👍

**Comment by seeziyang** [comment]
LGTM!

**Pull request merged**